client/src/components/Hero.js

```diff
@@ -1,10 +1,27 @@
 import { Flex, Heading } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
 
-export const Hero = ({ title }) => (
-  <Flex justifyContent="center" alignItems="center" height="100vh">
-    <Heading fontSize="10vw">{title}</Heading>
-  </Flex>
-);
+export const Hero = ({ title }) => {
+  const [subtitle, setSubtitle] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const baseUrl = process.env.NEXT_PUBLIC_API_URL;
+      fetch(baseUrl).then((data) => {
+        setSubtitle(`Request to backend :9090 status: ${data.status}`);
+        console.log(data);
```

Comment:
side note: shouldn't we avoid console.log and use a front-end logger

Reply:
You're right, let me remove that